frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-details-participant-list-table/common-details-participant-list-table.component.scss

```diff
@@ -0,0 +1,58 @@
+.participant-segment-table-container {
+  position: relative;
+  margin-top: 8px;
+  overflow: auto;
+  width: 100%;
+
+  .spinner {
+    position: sticky;
+    top: 0;
+    z-index: 1111;
```

Comment:
why 1111, just  1 would work... :sweat_smile:

Reply:
yeah, that's copied from existing implementations, but it's just that if other things later use z-index of 1 or 2 or 3 or 10, this should always be sure to be higher and on top.